src/components/purchasePanel.tsx

```diff
@@ -0,0 +1,50 @@
+import { useTranslations } from 'next-intl';
+import { Game } from '@/constants/types';
+import calculatePrice from '@/scripts/calculatePrice';
+
+function PurchasePanel({ selectedGame }: { selectedGame: Game }) {
```

Comment:
przenośmy zawsze to do deklaracji nad komponentem: `{ selectedGame: Game }` żeby pisać sobie ładnie interface